### PR TITLE
fix duplicated switch case in TestUpdateNodeObjects

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
'''
fix duplicated switch case in TestUpdateNodeObjects
				`case i%4 == 1:
					lastCount = 2
					n.Status.Conditions = []v1.NodeCondition{
						{
							Type:   v1.NodeReady,
							Status: v1.ConditionFalse,
							Reason: "foo",
						},
						{
							Type:   v1.NodeDiskPressure,
							Status: v1.ConditionTrue,
							Reason: "bar",
						},
					}
				case i%4 == 1:
					lastCount = 0
					n.Status.Conditions = nil

'''
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/integration/master/synthetic_master_test.go [hunk 1]
```diff
@@ -748,7 +748,7 @@ func TestUpdateNodeObjects(t *testing.T) {
 							Reason: "bar",
 						},
 					}
-				case i%4 == 1:
+				case i%4 == 2:
```
Comment (sttts) [inline test/integration/master/synthetic_master_test.go]: `==2` is really what we want here? This test is hard to follow.

Comment (hanxiaoshuai) [reply test/integration/master/synthetic_master_test.go]: So should I remove this 'case i%4 == 1 ' or else?

Comment (sttts) [reply test/integration/master/synthetic_master_test.go]: I don't know :) This test needs some higher level comments to explain what the goal is.
